fvm/src/machine/engine.rs

```diff
@@ -200,6 +202,31 @@ struct Cache<K> {
 }
 
 impl Engine {
+    /// Attempts to load Actor Code CID from the blockstore and
+    /// instantiate a wasmtime Module from the WASM bytecode if the
+    /// given code CID is present, otherwise returns None.
+    ///
+    /// Implicitly caches instantiated modules.
+    pub fn load_code_cid<BS: Blockstore>(
```

Comment:
Any reason why not to enhance get_module instead?

Reply:
yeah, that was my first attempt, but here we need access to the `blockstore`, which is accessible from the kernel. The kernel gets moved and consumed by value when crating a wasmer store. Then the store is needed to construct an `Instance`. This dependency cycle has few solutions, the least elegant and invasive is this thing that I have here.

Once I have a complete WASM prototype, I thought about tacking the lifetimes usage between `Kernel`, `CallManager` and `Machine`. I have few ideas around it but those changes would be quite big and touch many files.